src/Copy/BrandCopy.elm

```diff
@@ -14,7 +14,7 @@ brandCopy key =
 
         CallToActionOne ->
             CallToAction
-                { action = Phone
+                { action = DesktopInteractive
```

Comment:
Ah, I forgot this was a phone number here. If we don't want it to be clickable it will need to be a DesktopDisplayOnly